src/App.jsx

```diff
@@ -14,70 +16,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const [message, setMessage] = useState(`${selectedGood} is selected`);
```

Comment:
I don't think that it's a good idea to create a state associated with another state. Consider using useMemo instead. 
Or, otherwise, you should use useEffect to update the `message` state on each `selectedGood` update